### ISSUE TITLE
As a user I want to have subscriptions automatically created for a predefined set of parameters 

### ISSUE THREAD

**Issue by an-dream** [RHEAGROUP/COMET-IME-Community-Edition#796]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the COMET-IME
- [x] I have searched [open](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
It can be cumbersome to manually add subscriptions (single or batch) every time new Element Definitions with certain parameters are added. As a user I want to have subscriptions automatically created (for my domain) if certain parameters are present in the current engineering model.

A new subscriptions button to manage subscriptions could be envisioned. It should be possible to:

- Create a list of parameters to be automatically subscribed to (similarly to when you pick parameters to create Crossviews)
- Visualize and edit the whole list of others' parameters that my domain is subscribed to (i.e. my input parameters from others)
- Visualize and edit the whole list of my parameters that other domains are subscribed to (i.e. my output parameters to others)
- An N2 matrix (for the current domain and overall) to visualize all these relations.

Consider splitting this issue into more.

### Logs

  - [x] There are no logs regarding the issue
  - [ ] I have attached a log file or included the relevant section in this issue  
  
### System Configuration

- COMET version:
  - [x] COMET-IME: CDP4 7.0.0
  - [ ] Other: 
  -
- Web Services:
  - [ ] CDP4/COMET Web Services version: 
  - [ ] OCDT WSP version: 
- Environment (Operating system, version and so on):
  - [x] Microsoft Excel (32/64 bit):  Excel 365 MSO 16.0, 64 bit
  - [x] Windows OS (32/64 bit):  Windows 10, 64 bit
  - [ ] Additional information:

**Comment by samatrhea** [comment]
part of this already implemented in "batch subscriptions" where you can subscribe to all Parameters of a certain ParameterType and category applied. accessible via ElementDefinition browser -> right mouse click "create multiple subscriptions"

**Comment by an-dream** [comment]
Yes, the "batch subscription" functionality has been very helpful compared to right-clicking on each single parameter to subscribe.

However, every time a new block with parameters is added, the user has to go through all the list of parameters again in the "batch subscription", and re-apply the batch subscription again to cover the newly added parameters. So the idea could be to enhance this batch functionality with a dedicated "subscriptions manager" environment (accessible through a button in the model tab).

For example, if I'm a system engineer and I know I will use mass, powers (per mode), n_items, redundancy, etc. for my caluclations, I would like to set "default subscriptions", meaning that subscriptions are automatically created when a parameter of those is added by someone else in the model. This would speed up the subscription process considerably.
